src/components/welcome-page/Welcome.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import LinkedInIcon from '@material-ui/icons/LinkedIn';
+import WelcomeBtn from './WelcomeBtn';
+import './Welcome.scss';
+
+const Welcome: React.FC = () => {
+  const text = 'Linked';
+
+  const handleSignIn = () => {
+    console.log('sign in');
+  };
+  const handleSignUp = () => {
+    console.log('sign up');
+  };
+  return (
+    <div className="main-content">
```

Comment:
```suggestion
    <main className="welcome-page">
```

Reply:
main-content looks general - on other page it could also appear and it will cause styles overriding